### PR TITLE
service/tika: init

### PR DESCRIPTION
**Pull request by drupol**
Tika has been merged today (https://github.com/NixOS/nixpkgs/pull/279059) so we need to wait 2~3 days until it become available here.

Once there, I'll set this PR in Ready for review.

### CODE REVIEW

File: nix/services/tika_test.nix
Comment (shivaraj-bh) [inline nix/services/tika_test.nix:1]: Can you add this file to `test/flake.nix`?

**Comment by shivaraj-bh** [comment]
I am also planning to work on a script that will initialise all the files required to add a new service, but for now could you also add a `docs/tika.md` file with a brief description for `tika` and a `Getting started` block with `services.tika.enable = true;`? 

Further improvement on the docs can be done in a different PR upon coming up with use cases for using it with other services (I still have to update `searxng` docs as well, to use it with `open-webui`, I will do that soon)

File: nix/services/tika_test.nix
Comment (drupol) [reply nix/services/tika_test.nix:1]: Done

**Review by shivaraj-bh — APPROVED** [review]
Looks good! Only two things remaining:

1. Add a `docs/tika.md` file
2. Rewrite the commit messages to:
  - feat(tika): init
  - chore(searxng): use `http_get`

2 is because we use [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/#specification) to help during releases.

**Comment by drupol** [comment]
Done, should be good now.

**Comment by shivaraj-bh** [comment]
Thanks!

**Pull request merged**